docs/blog/2017-11-09-why-i-created-my-blog-with-gatsby-and-contentful/index.md

```diff
@@ -57,7 +57,7 @@ If you are a little bit more tech savvy and have experience provisioning
 software in a Linux distribution, you might prefer to rent a droplet from
 Digital Ocean and host the blog yourself. This, in turn, will allow you to have
 greater control over your site. However, you will also to pay for that droplet,
-even if it's the cheaper $5/month one.
+even if it's the cheaper \$5/month one.
```

Comment:
A bug in the new version of prettier apparently? We'd maybe _not_ upgrade if this is the case--until it's fixed.

Reply:
Although actually it seems to be displayed exactly as is (e.g. as if it wasn't escaped). I wonder _why_ this change was made 🤔

Reply:
Yeah I'm not 100% if this was intentional from prettier or not, can't find anything in the release notes about this other than https://prettier.io/blog/2018/11/07/1.15.0.html#preserve-math-syntax-5050-5220-by-ikatyang. Relevant issue https://github.com/prettier/prettier/issues/5483